gpkit/substitution.py

```diff
@@ -179,8 +178,7 @@ def substitution(nomial, substitutions, val=None):
                 sub = VarKey(name=sub, **descr)
                 exps_[i] += HashVector({sub: x})
                 varlocs_[sub].append(i)
```

Comment:
ok I think I get it -- string substitution is like a renaming operation? Create something just like `x` but with `.descr["name"]` overwritten by `"y"`?